segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/IndexAppend.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.segmentstore.server.host.handler;
+
+import io.pravega.shared.protocol.netty.ByteBufWrapper;
+import lombok.Data;
+
+/**
+ * Java class to manage index segment appends.
+ */
+@Data
+public class IndexAppend {
+    private final String indexSegment;
+    private final ByteBufWrapper bufferWrapper;
```

Comment:
Don't name members after their type. Anyone looking at this line already knows that, name it after what it means